Deprecate old OpenStack modules

emonty opened the pull request.

abadger commented:
Increasing priority slightly because I think we should decide whether to do this in time for the 1.9 release.

bcoca commented:
have we merged or do we have PR with new modules?

emonty committed:
Deprecate old OpenStack modules

There are a completely new set of modules that do all of the things like
keystone v3 and auth_plugins and the like correctly. Structurally
upgrading these would have been massively disruptive and there is no
real good way to do so without breaking people.

These modules should be kept around for several releases - they still
work for people - and they should get bug fixes. But they should not
take new features. New features should go to the os_ modules.

emonty commented:
@bcoca This is also now ready to land (I updated it to match the move of the modules into the openstack dir) I'd love to get this merged sooner rather than later to make it clear that these should really only be taking bugfixes
